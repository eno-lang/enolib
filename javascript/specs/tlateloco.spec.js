const fs = require('fs');
const path = require('path');

const { parse } = require('..');

const input = fs.readFileSync(path.join(__dirname, './tlateloco.eno'), 'utf-8');

describe('Tlateloco sample', () => {
  test('parses correctly', () => {
    const result = parse(input);

    expect(result.raw()).toMatchSnapshot();
  });
});
