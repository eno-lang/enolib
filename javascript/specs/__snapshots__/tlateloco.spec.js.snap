// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tlateloco sample parses correctly 1`] = `
Array [
  Object {
    "Titel": "Tlatelolco: Wohnblocks, Ruinen und Massaker",
  },
  Object {
    "Untertitel": "»Tlatelolco«, ein Film von Lotte Schreiber",
  },
  Object {
    "Bild": null,
  },
  Object {
    "Autoren": Array [
      "Georg Oswald",
    ],
  },
  Object {
    "Datum": "2013-01-01",
  },
  Object {
    "Sprache": "DE",
  },
  Object {
    "Kategorien": Array [
      "Besprechung",
      "Film",
    ],
  },
  Object {
    "Tags": Array [
      "Mexiko",
      "Tlatelolco",
      "Olympische Spiele",
      "Architektur",
      "Mexico City",
      "Polizeigewalt",
    ],
  },
  Object {
    "Permalink": "tlatelolco-wohnblocks-ruinen-und-massaker",
  },
  Object {
    "Buchbesprechungen": null,
  },
  Object {
    "Lesbar": "Ja",
  },
  Object {
    "Urbanize": null,
  },
  Object {
    "Abstract": null,
  },
  Object {
    "Text": "Der Stadtteil Tlatelolco ist ein Ort, ...
...Österreich, Mexiko, 75 min",
  },
  Object {
    "Literaturverzeichnis": null,
  },
]
`;
